src/components/BestBooks.js

```diff
@@ -43,6 +45,19 @@ export default class BestBooks extends Component {
     this.fetchBooks();
   }
 
```

Comment:
Feedback here: 

Try calling this method in `componentDidMount()` so data can be loaded after initial render etc.